samples/batch/csharp/batchclient/batchclient.webhooks.cs

```diff
@@ -0,0 +1,47 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+namespace BatchClient
+{
+    using System;
+    using System.Threading.Tasks;
+
+    public partial class BatchClient
```

Comment:
It would be easier for users to follow and copy-paste if each scenario (basic transcription w/o webhook, transcription with webhook) in samples is self-contained. But this can be done later.